ui/pages/confirm-transaction-base/gas-details-item/gas-details-item.scss

```diff
@@ -6,4 +6,12 @@
     color: $Grey-500;
     line-height: inherit;
   }
+
+  &__gasFeeLabel {
```

Comment:
Should we be using snake-case and not camelCase here? That seems to be a pattern across our CSS.

Reply:
Fixed